AUTHORS.md

```diff
@@ -23,6 +23,7 @@
 
 ## Contributors
 
-- Jeonghun Oh <jeonghunoh@korea.kr>
-- Knut Inge Skifjeld <knutinge@ndla.no>
-- Marc Von Opstdal <marc.van.opstal@ndla.no>
+- [Jeonghun Oh](https://github.com/totoromaum)
+- Knut Inge Skifjeld [(NDLA)](https://ndla.no)
```

Comment:
This will make the parenthesis around 'NDLA' a part of the link text, if only the text inside the parenthesis should be the link it should be `([NDLA](https://ndla.no))`